spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableProxyFactory.java

```diff
@@ -192,7 +192,7 @@ private boolean isPollable(Class<?> channelType) {
 	}
 
 	private void bridgeSubscribableToPollableChannel(SubscribableChannel sharedChannel, MessageChannel inputChannel) {
-		sharedChannel.subscribe(new MessageChannelBinderSupport.DirectHandler(inputChannel));
+		sharedChannel.subscribe(new DirectHandler(inputChannel));
 	}
 
 	private void bridgePollableToSubscribableChannel(MessageChannel pollableChannel,
```

Comment:
can these parameters be typed? (`PollableChannel` and `SubscribableChannel`, respectively)